cognitiveservices/textanalytics.go

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package cognitiveservices
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/services/cognitiveservices/v2.1/textanalytics"
+	"github.com/Azure/go-autorest/autorest"
+)
+
+func GetTextAnalyticsClient(apiEndPoint string, apiKey string) textanalytics.BaseClient {
+	textAnalyticsClient := textanalytics.New(apiEndPoint)
+	subscriptionKey := apiKey
+	textAnalyticsClient.Authorizer = autorest.NewCognitiveServicesAuthorizer(subscriptionKey)
+	return textAnalyticsClient
+}
+
+// returns a pointer to the string value passed in.
+func StringPointer(v string) *string {
```

Comment:
This is already available in `github.com/Azure/go-autorest/autorest/to.StringPtr()`